package.json

```diff
@@ -162,6 +162,7 @@
 		"terser-webpack-plugin": "^1.2.3",
 		"throng": "^4.0.0",
 		"tmp-promise": "^2.0.1",
+		"url-polyfill": "^1.1.5",
```

Comment:
So far we've been using polyfill.io and using a script tag within `server/Html.js`. This comes with the benefit that we don't load the polyfill into the bundle, and only fetch what's needed (polyfill.io returns a custom js based on the browser requesting it). Looks like polyfill.io has support for a URL polyfill.

Any reason you prefer using an npm import?

Reply:
Nope! I'll use the polyfill.io one instead.